plugins/legacy-preset-chart-nvd3/src/DistBar/controlPanel.ts

```diff
@@ -107,10 +112,24 @@ const config: ControlPanelConfig = {
     groupby: {
       label: t('Series'),
       validators: [validateNonEmpty],
+      mapStateToProps: (state, controlState) => {
+        const groupbyProps = sharedControls.groupby.mapStateToProps?.(state, controlState) || {};
+        groupbyProps.canDropValue = (column: ColumnMeta) =>
+          !ensureIsArray(state.controls.columns.value).includes(column.column_name);
```

Comment:
Question: Is it safe to assume the whole chain `state.controls.columns.value` is defined? Feels slightly safer to put `?` in between those nested properties..

Reply:
Done